src/importer/user-importer.js

```diff
@@ -1,35 +1,55 @@
 module.exports = function(Promise, importerFactory) {
-
   function prepare(connexion) {
-    return new Promise(function(resolve, reject) {
-      connexion.query('DELETE FROM user', function(err) {
-        if (err) {
-          return reject(err);
-        }
-        resolve();
+    return connexion.query('DELETE FROM users', { type: connexion.QueryTypes.DELETE })
+      .then(function() {
+        connexion.query('DELETE FROM user_custom_attributes', { type: connexion.QueryTypes.DELETE });
       });
-    });
   }
 
   function process(user) {
-    var companyId = user.companies.companies.length > 0 ? user.companies.companies[0].id : null;
-    return this.query('INSERT INTO user (id, user_id, name, email, pseudonym, city_name, continent_code, country_name, latitude, longitude, postal_code, region_name, timezone, country_code, last_request_at, last_seen_ip, created_at, signed_up_at, updated_at, session_count, user_agent_data, company_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', user.id, user.user_id, user.name, user.email, user.pseudonyme, user.location_data.city_name, user.location_data.continent_code, user.location_data.country_name, user.location_data.latitude, user.location_data.longitude, user.location_data.postal_code, user.location_data.region_name, user.location_data.timezone, user.location_data.country_code, user.last_request_at, user.last_seen_ip, user.created_at, user.signed_up_at, user.updated_at, user.session_count, user.user_agent_data, companyId).then(function() {
+    var connexion = this.connection,
+      companyId = user.companies.companies.length > 0 ? user.companies.companies[0].id : null,
+      query = this.query;
+
+    return query('INSERT INTO users (id, user_id, name, email, pseudonym, city_name, continent_code, country_name, latitude, longitude, postal_code, region_name, timezone, country_code, last_request_at, last_seen_ip, created_at, signed_up_at, updated_at, session_count, user_agent_data, company_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', {
+      type: connexion.QueryTypes.INSERT,
+      replacements: [user.id, user.user_id, user.name, user.email, user.pseudonyme, user.location_data.city_name, user.location_data.continent_code, user.location_data.country_name, user.location_data.latitude, user.location_data.longitude, user.location_data.postal_code, user.location_data.region_name, user.location_data.timezone, user.location_data.country_code, user.last_request_at, user.last_seen_ip, user.created_at, user.signed_up_at, user.updated_at, user.session_count, user.user_agent_data, companyId]
```

Comment:
Line is too long.<br>Identifier 'user_id' is not in camel case.<br>Identifier 'location_data' is not in camel case.<br>Identifier 'city_name' is not in camel case.<br>Identifier 'continent_code' is not in camel case.<br>Identifier 'country_name' is not in camel case.<br>Identifier 'postal_code' is not in camel case.<br>Identifier 'region_name' is not in camel case.<br>Identifier 'country_code' is not in camel case.<br>Identifier 'last_request_at' is not in camel case.<br>Identifier 'last_seen_ip' is not in camel case.<br>Identifier 'created_at' is not in camel case.<br>Identifier 'signed_up_at' is not in camel case.<br>Identifier 'updated_at' is not in camel case.<br>Identifier 'session_count' is not in camel case.<br>Identifier 'user_agent_data' is not in camel case.